src/components/Components/LinkedSecretsListView/LinkedSecretsListView.tsx

```diff
@@ -0,0 +1,173 @@
+import { LinkedSecretsListRow } from './LinkedSecretsListRow';
+
+const sortPaths: Record<SortableHeaders, string> = {
+  [SortableHeaders.secretName]: 'metadata.name',
+  [SortableHeaders.type]: 'type',
+};
+
+export const LinkedSecretsListView: React.FC = () => {
+  const { componentName, applicationName } = useParams<RouterParams>();
+  const namespace = useNamespace();
+  const applicationBreadcrumbs = useApplicationBreadcrumbs();
+  const [component, componentLoaded, componentError] = useComponent(namespace, componentName);
+  const [activeSortIndex, setActiveSortIndex] = React.useState<number>(SortableHeaders.secretName);
+  const [activeSortDirection, setActiveSortDirection] = React.useState<SortByDirection>(
+    SortByDirection.asc,
+  );
+  const [linkedSecrets, linkedSecretsLoaded, linkedSecretsError] = useLinkedSecrets(
+    namespace,
+    componentName,
+  );
+
+  const LinkedSecretsListHeader = React.useMemo(
+    () =>
+      getListHeader(activeSortIndex, activeSortDirection, (_, index, direction) => {
+        setActiveSortIndex(index);
+        setActiveSortDirection(direction);
+      }),
+    [activeSortDirection, activeSortIndex],
+  );
+
+  const sortedLinkedSecrets = useSortedResources(
+    linkedSecrets,
+    activeSortIndex,
+    activeSortDirection,
+    sortPaths,
+  );
+
+  if (!componentLoaded || !linkedSecretsLoaded) {
+    return (
+      <Bullseye>
+        <Spinner data-test="spinner" />
+      </Bullseye>
+    );
+  }
+
+  if (componentError || linkedSecretsError) {
+    const error = componentError ?? linkedSecretsError;
+    const httpError = HttpError.fromCode((error as { code: number }).code);
+    return (
+      <ErrorEmptyState
+        data-test="linked-secrets-list-view_error-empty-state"
+        httpError={httpError}
+        title="Unable to load linked secrets"
+        body={httpError?.message.length ? httpError?.message : 'Something went wrong'}
+      />
+    );
+  }
+
+  const NoDataEmptyMessage = () => (
+    <AppEmptyState
+      emptyStateImg={emptyStateImgUrl}
+      title="No linked secrets found"
+      data-test="linked-secrets-list-no-data-empty-message"
+    >
+      <EmptyStateBody>
+        To get started, select link secrets to link already existing secrets
+      </EmptyStateBody>
+      <Button
+        variant="primary"
+        // TODO: the "link secrets" functionality will be implemented in another ticket
+        // eslint-disable-next-line no-alert
+        onClick={() => alert('TODO')}
+        style={{ marginTop: 'var(--pf-v5-global--spacer--md)' }}
+      >
+        Link secrets
+      </Button>
+    </AppEmptyState>
+  );
+
+  return (
+    <>
+      <PageLayout
+        title="Manage linked secrets"
+        description={
+          <>
+            You can add new secrets to this component or unlink existing secrets.{' '}
+            {/* <ExternalLink href="">Learn more about linked secrets</ExternalLink> */}
```

Comment:
Whats with this comment?

Reply:
updated. We still don't have the link url, so I commented it out . I'll remove the comment for now, and once we have the link url a new ticket will be created to handle it :)